console.rst

```diff
@@ -342,6 +348,10 @@ method, which returns an instance of
 
     A new line is appended automatically when displaying information in a section.
 
+.. versionadded:: 6.2
+
+    Allow limiting the height of a console section was introduced in Symfony 6.2.
```

Comment:
Maybe the following sentence fits better?

The ability to limit the height....

Reply:
I tweaked the message a bit while merging. Thanks.